src/setup_node_env/without_babel_regster.js

```diff
@@ -17,13 +17,11 @@
  * under the License.
```

Comment:
As we discuss offline do you think we can rename that file to `prebuilt_dev_only_entry.js` in order to highlight the importance and the main purpose of it?